prow/config/presubmits.yaml

```diff
@@ -1245,6 +1245,55 @@ presubmits:
       - emptyDir: {}
         name: varlibdocker
 
+  - name: istio-push-images-2.3
+    trigger: (?m)^/test( | .* )push\-images,?($|\s.*)
+    rerun_command: /test push-images
+    skip_report: false
+    max_concurrency: 2
+    always_run: true
+    branches:
+    - ^maistra-2.3$
+    decorate: true
+    path_alias: istio.io/istio
+    labels:
+      preset-quay-pusher: "true"
+    spec:
+      containers:
+      - command:
+        - entrypoint
+        - sh
+        - -c
+        - HUB=quay.io/maistra TAG=PR-${PULL_NUMBER} make init push
+        env:
+        - name: GOFLAGS
+          value: -mod=vendor
+        - name: BUILD_WITH_CONTAINER
+          value: "0"
+        # FIXME: Uncomment this when we have our proxy in place
+        # - name: ISTIO_ENVOY_BASE_URL
+        #   value: https://storage.googleapis.com/maistra-prow-testing/proxy
+
+        # FIXME: Using upstream for now, replace with our image
+        # image: "quay.io/maistra-dev/maistra-builder:2.3"
+        image: "gcr.io/istio-testing/build-tools:release-1.14-0c4413d13921234efc115a6acc7097c47de0e815"
```

Comment:
Use our image

Reply:
BTW the other jobs are still using the upstream image. So, I think you're saying that we should modify them all to use our image?

Reply:
yes

Reply:
#310